crowbar_framework/lib/crowbar/installer.rb

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  class Installer
+    class << self
+      def install
+        crowbar_dir = Rails.root.join("..")
+
+        pid = spawn("sudo #{crowbar_dir}/bin/install-chef-suse.sh --crowbar")
```

Comment:
Somehow we need to check that we run on a SUSE system or not, since in theory, it's not the same script ;-) (I know, the other script is broken, but still)